.github/actions/regression-tests/action.yml

```diff
@@ -16,7 +16,12 @@ inputs:
   build-folder-name:
     required: true
     type: string
-
+  filter:
+    required: false
+    type: string
+  report-errors:
```

Comment:
@royjacobson I dont see you used it.. did you want to use it for the send notifications condition?

Reply:
yeah. now with the 'only on main' condition added it's probably redundant.